.github/workflows/oxcaml.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Install deps
         run: |
-          opam install . --deps-only
+          opam install . --deps-only --with-dev-setup
```

Comment:
This doesn't seem right? Why do you want `with-dev-setup` here?

Reply:
Isn't that where we define our test deps?

Reply:
I think that would be `--with-test`, but you don't need all of that just for oxcaml.